en/assembly/quick_start_durandal.md

```diff
@@ -0,0 +1,178 @@
+# Durandal Wiring Quick Start
+
+This quick start guide shows how to power the Holybro [Durandal](../flight_controller/durandal.md)<sup>&reg;</sup> flight controller and connect its most important peripherals.
+
+![Durandal](../../assets/flight_controller/durandal/durandal_hero.jpg)
+
+## Unboxing
+
+Durandal is sold bundled with a number of different combinations of accessories, including power modules: *PM02 V3* and *PM07*, and the *Pixhawk 4 GPS/Compass* (UBLOX NEO-M8N).
+
+The content of the box with the *PM02 V3* power module is shown below.
+This includes a pinout guide and power module instructions.
+
+![Durandal Box](../../assets/flight_controller/durandal/durandal_unboxing_1.jpg)
+
+The unpacked components, including cables, are:
+
+![Durandal - Fully Unboxed](../../assets/flight_controller/durandal/durandal_unboxing_2.jpg)
+
+
+## Wiring Chart Overview
+
+The image below shows how to connect the most important sensors and peripherals (except the motor and servo outputs).
+We'll go through each of these in detail in the following sections.
+
+![Durandal Wiring Overview](../../assets/flight_controller/durandal/durandal_wiring_overview.jpg)
+
+> **Tip** More information about available ports can be found here: [Durandal > Pinouts](../flight_controller/durandal.md#pinouts).
+
+
+## Mount and Orient Controller
+
+*Durandal* should be mounted on the frame using vibration-damping foam pads (included in the kit).
+It should be positioned as close to your vehicle’s center of gravity as possible, oriented top-side up with the arrow pointing towards the front of the vehicle.
+
+![Mounting/Orientation](../../assets/flight_controller/durandal/orientation.jpg)
+
+> **Note** If the controller cannot be mounted in the recommended/default orientation (e.g. due to space constraints) you will need to configure the autopilot software with the orientation that you actually used: [Flight Controller Orientation](../config/flight_controller_orientation.md).
+
+
+## GPS + Compass + Buzzer + Safety Switch + LED
+
+Durandal is designed to work well with the *Pixhawk 4 GPS module*, which has an integrated compass, safety switch, buzzer and LED.
+It connects directly to the [GPS port](../flight_controller/durandal.md#gps) using the 10 pin cable.
+
+<!-- Confirm how compass delivery works - separate or in the package? -->
+
+The GPS/Compass should be mounted on the frame as far away from other electronics as possible, with the direction marker towards the front of the vehicle (separating the compass from other electronics will reduce interference).
+
+![Connect compass/GPS to Durandal](../../assets/flight_controller/durandal/connection_gps_compass.jpg) 
+
+> **Note** The GPS module's integrated safety switch is enabled *by default* (when enabled, PX4 will not let you arm the vehicle).
```

Comment:
@bkueng This note comes from Pixhawk 4, which uses same GPS. Will it be true for Durandal?